scripts/cursor/rules-for-integration-templates.ts

```
@@ -0,0 +1,37 @@
+import { PERSONA_SECTION } from './shared.js';
+
+// File paths
+const GUIDES_DIR = './guides';
+const OUTPUT_DIR = './guides/rules-for-custom-nango-integrations';
+
+const PATHS = {
+    writing: `${GUIDES_DIR}/WRITING_SCRIPTS.md`,
+    templates: `${GUIDES_DIR}/WRITING_INTEGRATION_TEMPLATES.md`,
+    patterns: `${GUIDES_DIR}/ADVANCED_INTEGRATION_SCRIPT_PATTERNS.md`,
+    output: `${OUTPUT_DIR}/nango-best-practices.mdc`
+} as const;
```

ESLint

[error] Avoid casting objects. Add necessary type checks instead.

(@nangohq/custom-integrations-linting/no-object-casting)